template/tmpl/layout.tmpl

```diff
@@ -47,7 +47,6 @@
 	<div class="col-md-12">
 	<?js } ?>
 		<div id="main">
-    		<h1 class="page-title"><?js= title ?></h1>
```

Comment:
this is the fix from https://github.com/docstrap/docstrap/pull/89/files that should be here